frontend/src/scenes/plugins/InstalledPlugins.tsx

```diff
@@ -1,105 +1,46 @@
 import React from 'react'
-import { Button, Col, Row, Table, Tooltip } from 'antd'
+import { Button, Col, Row } from 'antd'
 import { useActions, useValues } from 'kea'
 import { pluginsLogic } from 'scenes/plugins/pluginsLogic'
-import { GithubOutlined, CheckOutlined, ToolOutlined, PauseOutlined } from '@ant-design/icons'
-import { PluginTypeWithConfig } from 'scenes/plugins/types'
+import { PlusOutlined } from '@ant-design/icons'
+import { PluginCard, PluginLoading } from './PluginCard'
 import { userLogic } from 'scenes/userLogic'
 
-function trimTag(tag: string): string {
-    if (tag.match(/^[a-f0-9]{40}$/)) {
-        return tag.substring(0, 7)
-    }
-    if (tag.length >= 20) {
-        return tag.substring(0, 17) + '...'
-    }
-    return tag
-}
```

Comment:
I think it's still good to show the `tag` somewhere. It's basically the "version" of the plugin. Right now it'll just be the SHA of the last master/main commit. However if we actually start tagging releases in plugins (via git tags), this will contain that information. We will also use this tag to compare to the latest tag from the server for updates.

Reply:
This can be pushed into the future for when we have the option to upgrade plugins, but just pointing it out now.